components/plurals/src/lib.rs

```diff
@@ -294,31 +296,113 @@ impl PluralRules {
     /// ```
     ///
     /// [`type`]: PluralRuleType
-    /// [`data provider`]: icu_provider::DataProvider
+    /// [`data provider`]: icu_provider
     pub fn try_new<T: Into<Locale>, D>(
         locale: T,
         data_provider: &D,
```

Comment:
You didn't rename any `data_provider` arguments to `resource_provider` when you changed the types. Why?

Reply:
Let's talk about naming in #1576